sys/net/include/socket_base/socket.h

```diff
@@ -24,7 +24,8 @@
 
 #include <stdint.h>
 
-#include "net/gnrc/ipv6.h"
+#include "net/af.h"
+#include "net/ipv6/addr.h"
```

Comment:
No `net/ipv6` was taken out of GNRC and is independent of it (it contains general definitions like addresses, headers and related functions). `net/ng_ipv6.h` was moved to `net/gnrc/ipv6.h` yesterday.